certora/applyMunging.patch

```diff
@@ -0,0 +1,108 @@
+diff -ruN DelegationToken.sol DelegationToken.sol
+--- DelegationToken.sol	2024-11-06 18:57:11.000000000 +0100
++++ DelegationToken.sol	2024-11-08 16:22:14.000000000 +0100
+@@ -1,17 +1,17 @@
+ // SPDX-License-Identifier: GPL-2.0-or-later
+ pragma solidity ^0.8.27;
+ 
+-import {IDelegation, Signature, Delegation} from "./interfaces/IDelegation.sol";
++import {IDelegation, Signature, Delegation} from "../../src/interfaces/IDelegation.sol";
+ 
+ import {ERC20PermitUpgradeable} from
+-    "../lib/openzeppelin-contracts-upgradeable/contracts/token/ERC20/extensions/ERC20PermitUpgradeable.sol";
++    "../../lib/openzeppelin-contracts-upgradeable/contracts/token/ERC20/extensions/ERC20PermitUpgradeable.sol";
+ import {ECDSA} from
+-    "../lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/utils/cryptography/ECDSA.sol";
++    "../../lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/utils/cryptography/ECDSA.sol";
+ import {Ownable2StepUpgradeable} from
+-    "../lib/openzeppelin-contracts-upgradeable/contracts/access/Ownable2StepUpgradeable.sol";
+-import {UUPSUpgradeable} from "../lib/openzeppelin-contracts-upgradeable/contracts/proxy/utils/UUPSUpgradeable.sol";
++    "../../lib/openzeppelin-contracts-upgradeable/contracts/access/Ownable2StepUpgradeable.sol";
++import {UUPSUpgradeable} from "../../lib/openzeppelin-contracts-upgradeable/contracts/proxy/utils/UUPSUpgradeable.sol";
+ import {ERC1967Utils} from
+-    "../lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/proxy/ERC1967/ERC1967Utils.sol";
++    "../../lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/proxy/ERC1967/ERC1967Utils.sol";
+ 
+ /// @title DelegationToken
+ /// @author Morpho Association
+@@ -44,6 +44,8 @@
+         mapping(address => uint256) _delegationNonce;
+     }
+ 
++    uint256 _zeroVirtualVotingPower = 0;
```

Comment:
Let's add a comment explaining what this variable is: it's a ghost variable, and it allows to track the voting power that the zero address would have (if it was not optimized out)

Reply:
Fixed in https://github.com/morpho-org/morpho-token-upgradeable/pull/92/commits/f80c7fbaebcbaf0f9b92f71aeafeed59a04c15fc.